<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Загрузка тарифов для Краснодарского края</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="6" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout2">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnImport">
       <property name="text">
        <string>Начать импортирование</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelNum">
       <property name="text">
        <string>всего записей в источнике:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Закрыть</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblSpr22">
     <property name="text">
      <string>Путь к Spr22</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="edtSpr22"/>
   </item>
   <item row="5" column="0" colspan="4">
    <widget class="QTextBrowser" name="log"/>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="btnSelect22">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QGroupBox" name="gbDublicates">
     <property name="title">
      <string>Совпадающие записи</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QRadioButton" name="chkUpdate">
        <property name="text">
         <string>Обновить</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="chkSkip">
        <property name="text">
         <string>Пропустить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="chkAskUser">
        <property name="text">
         <string>Спросить у пользователя</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="chkTFOMS">
     <property name="text">
      <string>Spr22 ТФОМС</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>chkTFOMS</sender>
   <signal>toggled(bool)</signal>
   <receiver>chkAskUser</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkTFOMS</sender>
   <signal>toggled(bool)</signal>
   <receiver>chkSkip</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkTFOMS</sender>
   <signal>toggled(bool)</signal>
   <receiver>chkUpdate</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
